services/user-actions.js

```diff
@@ -0,0 +1,15 @@
+import firebase from "firebase";
```

Comment:
Remember to remove this file. This goes beyond your PR's scope.

Reply:
i removed this line but got the error "cant find variable 'firebase'"